src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -171,6 +171,6 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('functionapp create', 'plan', options_list=('--plan', '-p'), completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                       help="name or resource id of the function app service plan. Use 'appservice plan create' to get one")
 register_cli_argument('functionapp create', 'consumption_plan_location', options_list=('--consumption-plan-location', '-c'),
-                      help="Geo location for functionsapps with consumption plans. Use 'functionapp list-consumption-locations' to see possible location plans")
+                      help="Geo location for functionsapps with consumption plans. Use 'functionapp list-consumption-locations' to see possible locations")
```

Comment:
I recommend being explicit in help commands and including the `az` command. FOr example, change this to ... 

```
help="Geo location for functionsapps with consumption plans. Use 'az functionapp list-consumption-locations' to see possible locations")
```

This allows people to copy and paste easily, and not guess.

Same would go for all places in help.